.github/workflows/libjs-test262.yml

```diff
@@ -107,3 +115,13 @@ jobs:
           repository-name: linusg/libjs-website
           token: ${{ secrets.BUGGIEBOT }}
           folder: libjs-website
+
+      - name: Compare non-bytecode
+        continue-on-error: true
+        working-directory: libjs-test262
+        run: ./per_file_result_diff.py -o master.json -n ../libjs-website/test262/master.json
```

Comment:
Please put them in https://github.com/linusg/libjs-website/tree/main/test262/data

Reply:
We might also want to clarify the names, e.g. `per-file-master.json`

Reply:
Yeah this one was missing the data folder,
I have added `per-file` to both of the names.